test/helpers.ts

```diff
@@ -476,8 +476,9 @@ export async function makeRewards(opts: RewardsOpts = {}): Promise<Rewards> {
   const rewards = await RewardsFactory.deploy(governor.address);
   await rewards.deployed();
 
-  for (const [comet, token] of configs) {
-    await wait(rewards.setRewardConfig(comet.address, token.address));
+  for (const [comet, token, multiplier] of configs) {
+    if (multiplier === undefined) await wait(rewards.setRewardConfig(comet.address, token.address));
+    else await wait(rewards.setRewardConfigWithMultiplier(comet.address, token.address, multiplier));
   }
```

Comment:
Should we read it back out here?

Reply:
Discussed offline. This is just a helper function to instantiate the rewards contract. The unit tests would be reading and asserting the values are set correctly.